RoslyJump.VSIX.2022/source.extension.vsixmanifest

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="RoslyJump.VSIX._2022.2b14b1a1-ba95-4ff2-8924-f961fd750512" Version="1.0" Language="en-US" Publisher="Pavel Sapehin" />
+        <DisplayName>RoslyJump.VSIX._2022</DisplayName>
```

Comment:
Display name should not contain `._2022`, currently:

```xml
 <DisplayName>RoslyJump.VSIX._2022</DisplayName>
```

Change it to `RoslyJump.2022`.

Reply:
Changed to `RoslyJump 2022`